lib/node_modules/@stdlib/ndarray/base/some-by/lib/main.js

```diff
@@ -0,0 +1,176 @@
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var iterationOrder = require( '@stdlib/ndarray/base/iteration-order' );
+var ndarray2object = require( '@stdlib/ndarray/base/ndarraylike2object' );
+var numel = require( '@stdlib/ndarray/base/numel' );
+var blockedaccessorsome2d = require( './2d_blocked_accessors.js' );
+var blockedaccessorsome3d = require( './3d_blocked_accessors.js' );
+var blockedsome2d = require( './2d_blocked.js' );
+var blockedsome3d = require( './3d_blocked.js' );
+var accessorsome0d = require( './0d_accessors.js' );
+var accessorsome1d = require( './1d_accessors.js' );
+var accessorsome2d = require( './2d_accessors.js' );
+var accessorsome3d = require( './3d_accessors.js' );
+var accessorsomend = require( './nd_accessors.js' );
+var some0d = require( './0d.js' );
+var some1d = require( './1d.js' );
+var some2d = require( './2d.js' );
+var some3d = require( './3d.js' );
+var somend = require( './nd.js' );
+
+
+// VARIABLES //
+
+var SOME = [
+	some0d,
+	some1d,
+	some2d,
+	some3d
+];
+var ACCESSOR_SOME = [
+	accessorsome0d,
+	accessorsome1d,
+	accessorsome2d,
+	accessorsome3d
+];
+var BLOCKED_SOME = [
+	blockedsome2d, // 0
+	blockedsome3d
+];
+var BLOCKED_ACCESSOR_SOME = [
+	blockedaccessorsome2d, // 0
+	blockedaccessorsome3d
+];
+var MAX_DIMS = SOME.length - 1;
+
+
+// MAIN //
+
+/**
+* Tests whether at least `n` elements in an ndarray pass a test implemented by a predicate function.
+*
+* ## Notes
+*
+* -   A provided ndarray should be an `object` with the following properties:
+*
+*     -   **dtype**: data type.
+*     -   **data**: data buffer.
+*     -   **shape**: dimensions.
+*     -   **strides**: stride lengths.
+*     -   **offset**: index offset.
+*     -   **order**: specifies whether an ndarray is row-major (C-style) or column major (Fortran-style).
+*
+* @param {ArrayLikeObject<Object>} arrays - array-like object containing an input ndarray and a 0-dimensional ndarray containing number of elements
+* @param {Function} predicate - predicate function
+* @param {thisArg} [thisArg] - predicate function execution context
+* @returns {boolean} boolean indicating whether `n` elements pass a test
+*
+* @example
+* var scalar2ndarray = require( '@stdlib/ndarray/from-scalar' );
+* var Float64Array = require( '@stdlib/array/float64' );
+*
+* function predicate( value ) {
+*    return value > 0.0;
+* }
+*
+* // Create a data buffer:
+* var xbuf = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 0.0, 7.0, 8.0, 9.0, 10.0, 11.0, 12.0 ] );
+*
+* // Define the shape of the input array:
+* var shape = [ 3, 1, 2 ];
+*
+* // Define the array strides:
+* var sx = [ 4, 4, 1 ];
+*
+* // Define the index offset:
+* var ox = 1;
+*
+* // Create the input ndarray-like object:
+* var x = {
+*     'ref': null,
+*     'dtype': 'float64',
+*     'data': xbuf,
+*     'shape': shape,
+*     'strides': sx,
+*     'offset': ox,
+*     'order': 'row-major'
+* };
+*
+* var n = scalar2ndarray( 3.0, { 'dtype': 'generic' } );
```

Comment:
```suggestion
* var n = scalar2ndarray( 3, {
*     'dtype': 'generic'
* });
```

Also, in JSDoc examples, we never put on a single line like this. We only do that in REPL help and TypeScript tests. This should be fixed here and anywhere else in this PR.